### DISCUSSION TITLE
PLZ REACT NATIVE WHAT I HAVE TO DO

### DISCUSSION THREAD

**Discussion by DEveL0perLuckY** [react-native-community/discussions-and-proposals#824] [Feedback]
autolinkLibrariesFromCommand: process cmd /c npx @react-native-community/cli config exited with error

**Comment by szymonrybczak** [comment]
hey @DEveL0perLuckY, could you please open issue inside https://github.com/react-native-community/cli repo with all necessary informations? :pray:

**Reply by DEveL0perLuckY** [reply]
https://github.com/react-native-community/cli/discussions/2526
plz can u review it 
https://github.com/react-native-community/cli/issues/2527
